Color contrast survey updates (#1961)
act-rules/act-rules.github.io: pages/glossary/highest-possible-contrast.md

objective: true
---

The highest value of the [contrast ratios](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) between two sets of colors (A and B).
To calculate the highest value, find the colors in each set with the largest and smallest 

> [relative luminance](https://www.w3.org/TR/WCAG21/#dfn-relative-luminance) (A-lum-max, A-lum-min, B-lum-max, B-lum-min). The

Reviewer: This is erroring. Can you add "luminance" to `__tests__/spelling-ignore.yml` so the tests pass?

Author: Done

> To calculate the highest value, find the colors in each set with the largest and smallest 
> [relative luminance](https://www.w3.org/TR/WCAG21/#dfn-relative-luminance) (`A-lum-max`, `A-lum-min`, `B-lum-max`, `B-lum-min`). The
> highest possible contrast is then the larger of the [contrast ratios](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio)
> of `A-lum-max` with `B-lum-min` versus `A-lum-min` with `B-lum-max`.

Reviewer 2: I think we can go in less details:
```suggestion
This is the highest [contrast ratios](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) between one color of A and one color of B.
```
I prefer to not be specific about how things should be computed, and only care about the result. Whether one actually compute all ratios for all pairs of colors, or is more clever and sort by luminance first, is an implementation detail which I'd rather leave out of the rules.
